generators/app/templates/src/containers/Register/register.component.js

```diff
@@ -100,15 +89,16 @@ class RegisterComponent extends Component<Props, State> {
                       target="_blank"
                       rel="noopener noreferrer"
                     >
-                      {t("register.whatIsAProvider")}
+                      {t('register.whatIsAProvider')}
                     </a>
                     <Link to="/login" className="a-with-spacing">
-                      {t("register.alreadySolid")}
+                      {t('register.alreadySolid')}
                     </Link>
                     <ul>
                       {providers.map((providerData, i) => (
                         <ProviderItem
                           data={providerData}
+                          // eslint-disable-next-line react/no-array-index-key
```

Comment:
Is there a way to remove the need for this disable?

Reply:
Fixed